Frontend/Gmfctn/src/app/modules/layout/layout.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subject, Subscription } from 'rxjs';
+import { take } from 'rxjs/operators';
+import { ProfileService } from 'src/app/core/services/profile-service/profile.service';
+import { Roles, User } from 'src/app/shared/models/user';
+import { defaultUser } from 'src/app/shared/models/dafault-user';
+
+@Component({
+  selector: 'app-layout',
+  templateUrl: './layout.component.html',
+  styleUrls: ['./layout.component.scss'],
+  providers: [ProfileService]
+})
+export class LayoutComponent implements OnInit, OnDestroy{
+  isSignedIn = false;
```

Comment:
It's not used anywhere as I see, so could be removed

Reply:
Fixed